e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -423,6 +426,112 @@ describe("scenarios > embedding > dashboard parameters with defaults", () => {
   });
 });
 
+describeEE("scenarios > embedding > dashboard appearance", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+    setTokenFeatures("all");
+  });
+
+  it("should not rerender the static embed preview unnecessarily (metabase#38271)", () => {
+    const textFilter = {
+      id: "3",
+      name: "Text filter",
+      slug: "filter-text",
+      type: "string/contains",
+    };
+
+    const dashboardDetails = {
```

Comment:
@npretto I added this filter, since I found out today that having a parameter will cause the appearance page preview to rerender every time we change any settings